back-end/Recognition/src/utils.py

```diff
@@ -0,0 +1,49 @@
+from google.cloud import vision
+
+def process_image(image_path):
```

Comment:
the function name is google_vision_call 😶

Reply:
1) also imagine that the function takes the photo as argument directly with type bytes  is name is photo:bytes